criu/net.c

```diff
@@ -1915,6 +1915,11 @@ static inline int dump_nftables(struct cr_imgset *fds)
 		return -1;
 
 	img = img_from_set(fds, CR_FD_NFTABLES);
+	img_fd = img_raw_fd(img);
+	if (img_fd < 0) {
+		pr_perror("Getting raw FD failed");
```

Comment:
you probably need to use pr_err in this case.

Reply:
fixed